app/controllers/integrations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class IntegrationsController < ApplicationController
+  load_and_authorize_resource
+
+  def update
+    if @integration.update_attributes(integration_params)
+      redirect_to installations_path(main_tab: 'integrations', integration_tab: @integration.provider), notice: 'Integration updated!'
+    end
+  end
+
+  def destroy
+    @integration.destroy
+    redirect_to installations_path(main_tab: 'integrations', integration_tab: @integration.provider), notice: 'Integration deleted!'
+  end
+
+  def create_task
+    begin
+      task = @integration.driver.create_task(params[:title])
+    rescue
+      redirect_to error_path(params[:error_id], task: task), flash: { error: "Operation failed!" }
+    end
+    redirect_to error_path(params[:error_id], task: task)
+  end
+
+  def notify_intercom_subscribers
```

Comment:
Why is this called `notify_intercom_subscribers`? We should make this smarter and utilize the notify_subscribers action and decide there if the user want's to use our system or intercom's. 

Also if we decide to keep this, we should not, we should not use names of drivers like this.